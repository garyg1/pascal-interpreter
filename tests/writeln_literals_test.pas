program writeln_literals_test;

var a : double;
var b : boolean;

const c = false;
const d = 4;

begin
    a := 5;
    b := true;

    Writeln(a);
    Writeln(b);
    Writeln(c);
    Writeln(d);
    Writeln('i am a string \literal');
    Writeln('escape sequence is double quote '' '' ');
    Writeln('');
    WRITELn();
end.