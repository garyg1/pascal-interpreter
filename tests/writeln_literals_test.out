5.0
true
false
4
i am a string \literal
escape sequence is double quote '' '' 


